skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -152,11 +147,8 @@ private function getCuratedContentButton() {
 
 	/**
 	 * Render default page header (with edit dropdown, history dropdown, ...)
-	 *
-	 * @param: array $params
-	 *    key: showSearchBox (default: false)
 	 */
-	public function executeIndex( $params ) {
+	public function index() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "index" is 32 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This function "index" has 244 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)